app/Http/Controllers/UsersController.php

```diff
@@ -67,12 +67,10 @@ public function search(Request $request)
         // Attempt to fetch all users.
         $users = $this->registrar->search($query);
 
-        if (! $users) {
-            return redirect()->route('users.index')->with('status', 'No user found!');
-        } elseif ($users->count() === 1) {
+        if ($users->count() === 1) {
             return redirect()->route('users.show', [$users->first()->id]);
         }
 
-        return view('users.search')->with(compact('users', 'query'));
+        return redirect()->route('users.index')->with('status', 'No user found!');
```

Comment:
Can we pass the empty `$users` collection to the view instead, and display an empty state there? I'm not sure if there are any scenarios where we would right now, but it'd be nice if we could display more than one result if there was one.

Reply:
@DFurnes right now, if a user is found, it automatically redirects to that user page (`/users/:northstar_id`). Should I change it to return the results in the grid format and then have the user who is searching click on the user they want?

Reply:
I feel like that's a kinda nice shortcut, unless it's been causing trouble for people.

Reply:
ok I think that should be fine! @ngjo is this ok if we change this?

Reply:
If we want to keep the current behavior, we could always just check for `$users->count() === 0` rather than `! $users` too. I'd just rather not rule out the possibility of having more than one search result.

Reply:
cool - chatted with @ngjo and since only results are returned that are an exact match, we'll stick with the user flow right now! I updated the above, @DFurnes !